source/index.txt

```diff
@@ -2,6 +2,18 @@
 MongoDB Connector for Spark
 ===========================
 
+.. toctree::
+   :titlesonly:
+
+   Get Started <getting-started>
+   Configure Spark <configuration>
+   Configure TLS/SSL <tls>
```

Comment:
To avoid repetition with previous entry

```suggestion
   TLS/SSL <tls>
```

Reply:
Left as is, because it feels wrong to leave out the word on the top level w/ no context otherwise